### ISSUE TITLE
Try to centralize strings parsing in get_expr_string

### ISSUE THREAD

**Issue by kinow** [common-workflow-lab/wdl-cwl-translator#185]
It could call the get_literal_value function, so the rest of the code can be centralized in the expr_string function.

See: https://github.com/common-workflow-lab/wdl-cwl-translator/pull/184#discussion_r807260870

**Comment by kinow** [comment]
Managed to debug the existing tests and compared the behaviour/output of `get_expr_string`, Expr.String.literal`, and `get_literal_value`.

![Screenshot from 2022-03-25 23-58-44](https://user-images.githubusercontent.com/304786/160108976-4655a1c3-fc8c-405e-b2b2-4accf74f99d6.png)

When the parts of the String expr are not of a certain type, then `literal` returns `None`. The rest of the code tries to create a JS expression, i think, to get a string. `get_literal_value` will recursively iterate the literal and create a value (no guarantee it's a string).

In summary, we cannot really use the same function to simplify/centralize it here. Closing! :+1:
